exo-go/features/run/app-dependency.feature

```diff
@@ -0,0 +1,23 @@
+Feature: running Exosphere applications with dependencies
```

Comment:
I fixed this in #333  but just wanted to point out that I got all of these from the original feature tests.